seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-mysql-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/MysqlCDCIT.java

```diff
@@ -0,0 +1,202 @@
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import static org.awaitility.Awaitility.await;
+
+@Slf4j
+@DisabledOnContainer(
+        value = {},
+        type = {EngineType.SPARK, EngineType.FLINK},
+        disabledReason = "")
+public class MysqlCDCIT extends TestSuiteBase implements TestResource {
+
+    private static final Logger LOG = LoggerFactory.getLogger(MysqlCDCIT.class);
+
+    // mysql
+    private static final String MYSQL_HOST = "mysql_cdc_e2e";
+    private static final String MYSQL_USER_NAME = "st_user";
+    private static final String MYSQL_USER_PASSWORD = "seatunnel";
+    private static final String MYSQL_DATABASE = "mysql_cdc";
+    private static final MySqlContainer MYSQL_CONTAINER = createMySqlContainer(MySqlVersion.V8_0);
+
+    private final UniqueDatabase inventoryDatabase =
+            new UniqueDatabase(MYSQL_CONTAINER, MYSQL_DATABASE, "mysqluser", "mysqlpw");
+
+    // mysql statement
+    private static final String SOURCE_SQL = "select * from mysql_cdc.products_cdc";
+    private static final String SINK_SQL = "select * from mysql_cdc.products_cdc_sink";
+
+    private static MySqlContainer createMySqlContainer(MySqlVersion version) {
+        MySqlContainer mySqlContainer =
+                new MySqlContainer(version)
+                        .withConfigurationOverride("docker/server-gtids/my.cnf")
+                        .withSetupSQL("docker/setup.sql")
+                        .withNetwork(NETWORK)
+                        .withNetworkAliases(MYSQL_HOST)
+                        .withDatabaseName(MYSQL_DATABASE)
+                        .withUsername(MYSQL_USER_NAME)
+                        .withPassword(MYSQL_USER_PASSWORD)
+                        .withLogConsumer(new Slf4jLogConsumer(LOG));
+        return mySqlContainer;
+    }
+
+    @BeforeAll
+    @Override
+    public void startUp() throws ClassNotFoundException, InterruptedException {
+        LOG.info("The second stage: Starting Mysql containers...");
+        Startables.deepStart(Stream.of(MYSQL_CONTAINER)).join();
+        LOG.info("Mysql Containers are started");
+        inventoryDatabase.createAndInitialize();
+        LOG.info("Mysql ddl execution is complete");
+    }
+
+    @TestTemplate
+    public void test(TestContainer container) throws IOException, InterruptedException {
+        LOG.info("-------docker mysql host:{}", MYSQL_CONTAINER.getHost());
+        LOG.info("-------docker mysql port:{}", MYSQL_CONTAINER.getDatabasePort());
+        LOG.info("-------docker mysql database:{}", MYSQL_CONTAINER.getDatabaseName());
+
+        CompletableFuture<Void> executeJobFuture =
+                CompletableFuture.supplyAsync(
+                        () -> {
+                            try {
+                                container.executeJob("/mysqlcdc_to_console.conf");
```

Comment:
rename `mysqlcdc_to_console.conf` to `mysqlcdc_to_jdbc.conf`
```suggestion
                                container.executeJob("/mysqlcdc_to_jdbc.conf");
```